addons/api/stringtable.xml

```diff
@@ -0,0 +1,179 @@
+        </Key>
+        <Key ID="STR_ACRE_api_basicMissionSetup_BabelSetup_No">
+            <English>No Babel</English>
+            <French>Ne pas utiliser Babel</French>
+        </Key>
+        <Key ID="STR_ACRE_api_basicMissionSetup_DefaultRadio_DisplayName">
+            <English>Default Radio 1</English>
+            <French>Radio par défaut 1</French>
+        </Key>
+        <Key ID="STR_ACRE_api_basicMissionSetup_DefaultRadio_Two_DisplayName">
+            <English>Default Radio 2</English>
+            <French>Radio par défaut 2</French>
+        </Key>
+        <Key ID="STR_ACRE_api_basicMissionSetup_DefaultRadio_Three_DisplayName">
+            <English>Default Radio 3</English>
+            <French>Radio par défaut 3</French>
+        </Key>
+        <Key ID="STR_ACRE_api_basicMissionSetup_DefaultRadio_Four_DisplayName">
+            <English>Default Radio 4</English>
+            <French>Radio par défaut 4</French>
+        </Key>
+        <Key ID="STR_ACRE_api_basicMissionSetup_DefaultRadio_Description">
+            <English>Default radio for ACRE to give player</English>
+            <French>Radio ACRE par défaut à donner au joueur.</French>
+        </Key>
+        <!-- Name Channels Module-->
+        <Key ID="STR_ACRE_api_nameChannels_Module_DisplayName">
+            <English>Name Channels</English>
+            <French>Nom des canaux</French>
+        </Key>
+        <Key ID="STR_ACRE_api_nameChannels_Module_Description">
+            <English>This module defines basic channel naming for a mission in ACRE</English>
+            <French>Ce module permet de définir le nommage de base des canaux radio ACRE pour la mission.</French>
+        </Key>
+        <Key ID="STR_ACRE_api_nameChannels_SideSelect_DisplayName">
+            <English>Side</English>
+            <French>Faction(s)</French>
+        </Key>
+        <Key ID="STR_ACRE_api_nameChannels_SideSelect_Description">
+            <English>Select the side to name channels for</English>
+            <French>Choisir pour quelle(s) faction(s) nommer les canaux.</French>
+        </Key>
+        <Key ID="STR_ACRE_api_nameChannels_SideSelect_All">
+            <English>All</English>
+            <French>Toutes</French>
+        </Key>
+        <Key ID="STR_ACRE_api_nameChannels_SideSelect_West">
+            <English>West</English>
+            <French>Ouest</French>
+        </Key>
+        <Key ID="STR_ACRE_api_nameChannels_SideSelect_East">
+            <English>East</English>
+            <French>Est</French>
+        </Key>
+        <Key ID="STR_ACRE_api_nameChannels_SideSelect_Ind">
+            <English>Independent</English>
+            <French>Indépendants</French>
+        </Key>
+        <Key ID="STR_ACRE_api_nameChannels_SideSelect_Civ">
+            <English>Civilian</English>
+            <French>Civils</French>
+        </Key>
+        <Key ID="STR_ACRE_api_nameChannels_Channel_1_DisplayName">
+            <English>Channel 1</English>
```

Comment:
Strings like that can be `Channel %1` and in SQF `format [LLSTRING(...), 1]`.

Not sure if that works for config though.

Reply:
Thank you for your approval, @jonpas :) !
However, I am really a beginner ; I would like to try what you just suggested, but I don't know how to do it at all.

Could you teach me, please :) ?

Reply:
@jonpas 
I try the following code in the _CfgModules.hpp_ file :
```            
class Channel_1 {
    displayName = format [LLSTRING(nameChannels_ChannelNumber), 1];
    description = CSTRING(nameChannels_Channel_1_Description);
    defaultValue = "PLTNET 1";
};
```
With that in the _stringtable.xml file_ :
```
<Key ID="STR_ACRE_api_nameChannels_ChannelNumber">
    <English>Channel %1</English>
    <French>Canal %1</French>
</Key>
```
But unfortunately, it's not found :( .

Reply:
Disregard, doesn't work in configs.